core/src/main/java/org/kohsuke/stapler/export/Property.java

```diff
@@ -147,18 +156,11 @@ public void writeTo(Object object, TreePruner pruner, DataWriter writer) throws
         }
     }
 
-    private Object safeGetValue(Object o) throws IOException {
-        try {
-            return getValue(o);
-        } catch (IllegalAccessException e) {
-            IOException x = new IOException("Failed to write " + name);
-            x.initCause(e);
-            throw x;
-        } catch (InvocationTargetException e) {
-            IOException x = new IOException("Failed to write " + name);
-            x.initCause(e);
-            throw x;
-        }
+    /**
+     *  Ignores property if {@link ExportInterceptor#getValue(Property, Object)} throws NotExportableException.
```

Comment:
Huh? It does not seem to do anything of the sort. In fact why does this method exist? Inline it!